README.md

```diff
@@ -730,7 +768,6 @@ that you get for free from OverReact, you're ready to start building your own cu
 * #### Stateful Component Boilerplate
 
     ```dart
-    import 'dart:html';
     import 'package:over_react/over_react.dart';
     part 'foo_component.over_react.g.dart';
 
```

Comment:
Ooh if you're in here editing this, we probably want to add the ignore comments to factories
```diff
UiFactory<BarProps> Bar = _$Bar; // ignore: undefined_identifier
```